forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -112,6 +112,7 @@ public class Card extends GameEntity implements Comparable<Card>, IHasSVars {
     private Map<Long, CardCollection> mustBlockCards = Maps.newHashMap();
     private List<Card> blockedThisTurn = Lists.newArrayList();
     private List<Card> blockedByThisTurn = Lists.newArrayList();
+    private Map<Player, CardCollection> chosenMap = Maps.newHashMap();
```

Comment:
We need to think if this can just replace chosenCards but if it needs to store List<Card> instead for _Human-Time Lord Meta-Crisis_ to work correctly